API/DTOs/LoginDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace API.DTOs
+{
+    public class LoginDto
+    {
+        public string Email { get; set; }
```

Comment:
Does this class have warnings? If so use `public string Email { get; set; } = default!`.